humble/distribution.yaml

```diff
@@ -2623,6 +2623,21 @@ repositories:
       url: https://github.com/humanoid-path-planner/hpp-fcl.git
       version: devel
     status: developed
+  hri_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/hri_msgs.git
+      version: humble-devel
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros4hri/hri_msgs-release.git
```

Comment:
If you intend to support releasing in ROS 2 Rolling in the future we'll want to move this release repository to the [ros2-gbp](https://github.com/ros2-gbp). You can request that [here](https://github.com/ros2-gbp/ros2-gbp-github-org/issues/new/choose).

Reply:
ok, thanks. I'm not yet 100% familiar with the ROS 2 release process...! Following https://docs.ros.org/en/humble/How-To-Guides/Releasing/First-Time-Release.html I've created an issue here to create a 'ros4hri' release team (https://github.com/ros2-gbp/ros2-gbp-github-org/issues/439), but  it is not very clear from this documentation what is the role of this release team.

Anyway, happy to move the release repo to `ros2-gbp` if that's better (how does it impact ROS 1 releases, though? should I maintain 2 release repos, one for ROS 1, one for ROS 2?)